crates/fuel-gas-price-algorithm/src/lib.rs

```diff
@@ -4,3 +4,24 @@
 
 pub mod v0;
 pub mod v1;
+
+#[allow(clippy::cast_possible_truncation)]
+pub(crate) fn cumulative_percentage_change(
```

Comment:
It feels unclean to have this function on the crate root. How about putting it in a `utils` module or something similar?

Reply:
Updated in https://github.com/FuelLabs/fuel-core/pull/2219/commits/d1672d32199c4d29da6c2430bfef9f1d47937e8c